monero-outreach-docs/translations/fr/PSA/files/learning/learning_fr.md

```diff
@@ -0,0 +1,18 @@
+# Cherchez à en Savoir Plus
+
+Au moment où nous écrivons ces lignes, une hypothèse commune dit qu’une grande majorité de la communauté Monero est composée de pionniers qui ont une plus grande maîtrise technologique que le commun des mortels. Le code de Monero découle du travail de quelques centaines de développeurs, ce qui est bien plus que ce que peuvent revendique la plupart des autres projets en crypto. Certains avancent que l’une des raisons pour laquelle le prix du Monero ne s’est pas plus rapproché du Bitcoin est car la technologie de Monero est compliquée à comprendre et à apprécier par les profanes. Mais nous, pionniers, continuons d’avancer dans l’espoir que Monero soit largement reconnu et adopté. Monero dans 100 ans dépendra toujours des personnes, comme vous, oeuvrant à la sauvegarde de son écosystème et améliorant encore le code. Ce sont toutes les raisons pour lesquelles nous recommandons que les personnes devraient toujours chercher à en savoir plus, plus particulièrement si vous n’êtes pas un développeur. Vous pouvez renforcer votre « alphabétisation Monero » en absorbant le plus d’informations que vous pourrez.
+
+En en apprenant plus sur les fonctionnalités de Monero, le code, l’idéologie open-source Cypherpunk qui l’a engendré et les méthodes pour améliorer la sécurité et la confidentialité en l’utilisant, nous pourrons ensemble rendre l’écosystème Monero encore plus fort. En ayant une bonne maîtrise de tous les aspects de Monero, vous serez également capable d’en parler à d’autres et favoriserez une plus grande adoption. Vous serez plus à l’aise en stockant, utilisant, et faisant des transactions XMR. La connaissance, c’est le pouvoir et elle peut changer le monde. Cherchez toujours à en savoir plus... et faites vos propres recherches !
+
+### Le future commence avec vous, voici quelques lectures pour vous aider à en apprendre plus sur Monero :
+
+**_[getmonero.org/fr (Site Web Officiel)](https://getmonero.org/)_**
```

Comment:
It should look like this:

getmonero.org (Site Web Officiel)](https://www.getmonero.org/fr/)

[getmonero.org (Site Web Officiel)](https://www.getmonero.org/fr/)

getmonero.org is the official site and when people reading the French version when clicking will be forward to the 'fr' version.

Reply:
Yes, my fault XD